source/pug/partials/events.pug

```diff
@@ -4,7 +4,9 @@
     h4 Launch events
 
 .container
-  include:markdown ../../markdown/events.md
+  .row.justify-content-center
+    .col-12.col-xl-10
```

Comment:
Sorry, I didn't give enough detail. This was in response to [this issue](https://github.com/MozillaFoundation/NSF-wireless-innovation-challenge/issues/28) to show @ScottDowne how to incorporate bootstrap here to reduce the width of the paragraph, rather than trying to deal with the `.container` class. Feel free to close if you're taking a different approach for that issue.